### DISCUSSION TITLE
Displaying Cards in Chat

### DISCUSSION THREAD

**Discussion by LucifurMacomb** [fvtt-fria-ligan/twilight2000-foundry-vtt#148] [Q&A]
Hi ! 

Was looking at running a game in this system digitally for the first time and noticed that there is a screenshot of a card being drawn from a deck displayed in chat; I have tried to recreate this and am wondering if there is something I am missing? I am aware there have been modules for initiative cards to be drawn, and am curious if this is a feature I am missing, a module/macro, or possible created for promotional material? Any insight would be sincerely appreciated, thanks! 

Screenshot in question: [https://cdn.shopify.com/s/files/1/2129/8201/files/FoundryVTT_2021-08-14_00-17-10.jpg?v=1685348121](Image)

**Comment by Stefouch** [comment]
Hi !

It's done with the "`Draw Encounter Card`" macro, which script is:

```js
await game.t2k4e.generators.encounter();
```

PS: Macro "`Roll On Table`" works too.
